tests/foreman/ui/test_activationkey.py

```diff
@@ -795,7 +793,68 @@ def test_positive_update_activation_key_4(self, cv2_name):
                                  (common_locators["alert.success"]))
             selected_cv = self.activationkey.get_attribute(name, cv_locator)
             self.assertEqual(cv2_name, selected_cv)
-            # TODO: Need to check for RH Product too
+
+    @run_only_on('sat')
+    @data(*valid_data_list())
+    def test_positive_update_activation_key_with_rh_product(self, cv2_name):
+        """@Test: Update Content View in an Activation key
+
+        @Feature: Activation key - Positive Update
+
+        @Steps:
+        1. Create Activation key
+        2. Update the Content view with another Content view
+        which has RH products
```

Comment:
The current text is syntactically wrong. Try checking it against a reST compiler, such as the online one [here](http://rst.ninjs.org/).